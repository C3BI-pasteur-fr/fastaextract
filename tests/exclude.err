file test.fa: Extracted 1 from 4 sequences (25.00 %)
