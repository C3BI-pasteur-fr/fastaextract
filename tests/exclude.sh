#! /bin/sh

### Set verbose mode
test "x$VERBOSE" = "xx" && set -x

#exclude without list
../src/faextract -x -o tmp.out $srcdir/test.fa 2> tmp.err || exit 1
diff $srcdir/fullexclude.out tmp.out || exit 1
diff $srcdir/fullexclude.err tmp.err || exit 1

#extract with list
../src/faextract -x -o tmp.out -l $srcdir/id_list.txt $srcdir/test.fa 2> tmp.err || exit 1
diff $srcdir/exclude.out tmp.out || exit 1
diff $srcdir/exclude.err tmp.err || exit 1


exit 0

