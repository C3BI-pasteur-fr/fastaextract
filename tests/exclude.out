>seq4 proteic
HRCYYWTTYGWBKRRTHNRUSBTWKTKRYDUNTGSTNWNHNMYNCAVHCUKRKSVYMTVCTSVGWHKKWTYTKTNSKUSD
HTRBNKGDKAMUYWHACSHGRKWMSWRWUTGBDMBSRMUR
